go/libkb/pgp_key.go

```diff
@@ -394,24 +394,30 @@ func finishReadOne(lst []*openpgp.Entity, armored string, err error) (*PGPKeyBun
 		}
 	}
 
+	if g != nil {
+		for _, bs := range first.Entity.BadSubkeys {
+			g.Log.Warning("Bad SubKey error: %v", bs.Err)
+		}
+	}
```

Comment:
@maxtaco are there any other warnings we might want to log, that don't correspond to a bad subkey? This was all that I thought of.